gcam-usa.md

```diff
@@ -7,11 +7,11 @@ gcam-version: v5.3
 The Global Change Assessment Model (GCAM) and GCAM-USA
 ------------------------------------------------------
 
-The GCAM model was expanded to include greater spatial detail in the USA region, referred to as GCAM-USA. In GCAM-USA the 50 U.S. states plus the District of Columbia (hereafter, the 51 states) are included as explicit regions that operate within the global GCAM model. Energy transformation (electricity generation and refined liquids production) and end-use demands (buildings, transportation, and industry) are modeled at the individual state level. Inter-state trade of all energy goods is considered, with state-specific consumer price mark-ups assigned for coal, natural gas, and refined liquids assigned based on price data from EIA (2012).
+The GCAM model was expanded to include greater spatial detail in the USA region, referred to as GCAM-USA. In GCAM-USA the 50 U.S. states plus the District of Columbia (hereafter, the 51 states) are included as explicit regions that operate within the global GCAM model. Energy transformation (electricity generation and refined liquids production) and end-use demands (buildings, transportation, and industry) are modeled at the individual state level. Inter-state trade of all energy goods is considered, with state-specific consumer price mark-ups assigned for coal, natural gas, and refined liquids assigned based on price data from [EIA 2017b](gcam-usa.html#eia2017b).
 
 <img src="gcam-figs/gcam-usa.png" width="397" height="525" />
 
-Note that several aspects of the energy system are still modeled at the aggregate U.S. level. Most notably, this applies to primary production of fossil resources including oil, gas, and coal. The supply of biomass energy feedstocks, which include residues and dedicated energy crops, is modeled at the level of 10 agro-ecological zones (AEZ) in the United States (Calvin et al 2014, Wise et al 2014). As with primary fossil resources, biomass feedstocks are assumed to be freely traded at one global price.
+Note that several aspects of the energy system are still modeled at the aggregate U.S. level. Most notably, this applies to primary production of fossil resources including oil, gas, and coal. The supply of biomass energy feedstocks, which include residues and dedicated energy crops, is modeled at the level of 22 water basins in the United States ([Calvin et al. 2019](gcam-usa.html#calvin2019), [Calvin et al. 2014](gcam-usa.html#calvin2014), [Wise et al. 2014](gcam-usa.html#wise2014)). As with primary fossil resources, biomass feedstocks are assumed to be freely traded at one global price.

```

Comment:
I think this last sentence needs to be updated. Starting with GCAM v5.2 we have regional markets for biomass, so the feedstocks are traded but not at one global price.